csbpg/data_owner_role.go

```diff
@@ -4,37 +4,31 @@ import (
 	"database/sql"
 	"fmt"
 	"log"
-	"sync"
 
 	"github.com/lib/pq"
 )
 
-var sharedDataOwnerRoleCreateMutex sync.Mutex
```

Comment:
@tinygrasshopper added this because he did a test which tried to create two binding at the same time and hit a fatal DB lock.

Reply:
Yes, I moved the mutex to the binding creation method, as now everything is created inside a transaction, and it no longer makes sense here.